src/solc_0.6/EstateSale/AuthValidator.sol

```diff
@@ -11,6 +11,9 @@ contract AuthValidator is Admin {
     event SigningWallet(address indexed signingWallet);
 
     constructor(address adminWallet, address initialSigningWallet) public {
+        require(adminWallet != address(0), "AuthValidator: zero address");
+        require(initialSigningWallet != address(0), "AuthValidator: zero address");
```

Comment:
you're checking twice `initialSigningWallet` since it's done in `_updateSigningAuthWallet` too

Reply:
We need to have on both: 1 to validate the argument passed to the constructor. And the 2nd one is to valide when the signing wallet is being updated under updateSigningAuthWallet(), which is an external call.